### PR TITLE
libmysqlclient

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
I have a doubt about versioning. The library files are 18.3.0 but the source tarball it's 6.1.6.

https://codein.withgoogle.com/dashboard/task-instances/6362508340232192/?sp-page=1

### CODE REVIEW

File: dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+BUILD()
+{
+	cmake . -DCMAKE_INSTALL_PREFIX=$prefix -DINSTALL_BINDIR=$relativeBinDir \
+		-DINSTALL_INCLUDEDIR=$relativeIncludeDir -DINSTALL_LIBDIR=$relativeLibDir \
+		-DINSTALL_MANDIR=$relativeManDir -DINSTALL_SHAREDIR=$relativeDataDir \
+		-DINSTALL_MYSQLDATADIR=$relativeDataDir -DINSTALL_MYSQLSHAREDIR=$relativeDataDir \
+		-DWITH_EXTRA_CHARSETS=all -DCOMPILATION_COMMENT="Haiku" \
+		-DSYSTEM_TYPE="Haiku"
+	make
```
Comment (waddlesplash) [inline dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe]: Add `$jobArgs`?

**Comment by pulkomandy** [comment]
The library version and package version are different things. So you should use 6.1.6 for the package, and 18.3.0 for the lib: and devel: provides.
About the patch: you have several whitespace changes, it would be nice to avoid those as they make it harder to read the changes (try to have a patch as small as possible). Also, there are changes that will break the build or remove features on other systems, which is not acceptable. Instead of deleting code, add ````#ifndef __HAIKU__```` or similar guards.

**Comment by AdrianArroyoCalle** [comment]
Updated for $jobArgs

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**